### DISCUSSION TITLE
reset rdkit?

### DISCUSSION THREAD

**Discussion by rwst** [rdkit/rdkit#4439] [Q&A] [answered]
Hello,
is there a way to make sure that rdkit state is reset? I'm asking because I see here that matching may be affected by internal state but, before I start hunting, I want to apply any simple solution.

**Comment by greglandrum** [comment, marked as answer]
@rwst : I'm not quite sure what you mean, but I don't believe that substructure matching is affected by any internal state aside from the arguments passed to the substructure matching function. Have you seen behavior which leads you to believe that there is some form of internal state?

**Reply by rwst** [reply]
This one turned out to be my error but it's good to know that my hypothesis would have had low probability.

Thanks.
